05-requests/new-ticket.yaml

```diff
@@ -0,0 +1,142 @@
+id: new-ticket
+type: PageHeaderMenu
+
+requests:
+  - id: save_new_ticket
+    type: KnexRaw
+    connectionId: knex
+    payload:
+      _state: true
+    properties:
+      query: |
+        INSERT INTO tickets (
```

Comment:
I think it might be better to use named bindings instead of positional bindings to promote this as a better practice: https://knexjs.org/guide/raw.html#raw-parameter-binding

Reply:
Changes made in ae20a454666402b2b3f545a9f05ef7985c7f50e4